integration-cli/docker_cli_info_test.go

```diff
@@ -146,3 +146,21 @@ func (s *DockerSuite) TestInfoDisplaysStoppedContainers(c *check.C) {
 	c.Assert(out, checker.Contains, fmt.Sprintf(" Paused: %d\n", 0))
 	c.Assert(out, checker.Contains, fmt.Sprintf(" Stopped: %d\n", 1))
 }
+
+func (s *DockerSuite) TestInfoDebug(c *check.C) {
```

Comment:
Although, if we don't already have a daemon-side test verifying the `--debug` shows up in the "info" output then keeping this test for that would be good too  :-)